guides-and-standards/guide-choosing-software-dependencies.md

```diff
@@ -0,0 +1,62 @@
+- **The tool is easily installable.** To be able to use a tool, we need to be able to install it. Below I’ve included somewhat arbitrary gradations of what makes something easy to install. You can check these things by eye to quickly see if you think a tool will be installable, but often the best way to determine this is to install and use the tool.
+  - Very easy: cross-platform installation as a conda package
+  - Easy: installable from another package repository with another package manager like pip or poetry (python) or `install_packages()` (R)
+  - Somewhat easy: Installable from GitHub with a package manager
+  - Somewhat easy: Has a dockerfile (a recipe from which a Docker container can be built)
+  - Somewhat difficult: has an environment yaml file or requirements.txt file that lists versioned dependencies and has installation instructions that involve cloning the repository
+  - Difficult: git clone & compile from source
+  - Difficult: No installation instructions
+- **The software is maintained.**
+  - Hint: there is a difference in the timestamp between the first and last commit and the last commit occurred recently (~months)
+  - Hint: there are responses to issues
+  - Hint: look for a history of closed Pull Requests
+  - Hint: look at the "insights" page on GitHub. The "contributors" tab will show contributions over time
+- **Other people use the software.** This signals that other people believe in the method and that other people were able to install and use the software.
+  - Hint: the GitHub repository has a lot of stars
+  - Hint: the paper publishing the software has been cited frequently by other projects that used the tool.
+
+### Criteria that are nice to have (but not necessary if you’re willing to do the leg work to make sure the tool works)
+
+- **The tool has tests.** Tests ensure the accuracy and quality of a software tool. See [this AUG](https://training.arcadiascience.com/arcadia-users-group/20231104-testing-concepts/lesson/) for more about software tests.
+  - Hint: tests will often be a `test/` or `tests` folder, either in the root of the repository or
```

Comment:
I'm unclear if this `or` is supposed to contain more, or if it's a segue to the next bullet point. It seems odd to have it be to introduce the next section since it's under this specific hint, but maybe that's just me.

Reply:
oh nope, this is just an unfinished sentence that snuck through. Thank you for catching this!